src/client/Inode.h

```diff
@@ -211,7 +211,8 @@ struct Inode : RefCountedObject {
   int cache_gen = 0;
   int snap_caps = 0;
   int snap_cap_refs = 0;
-  utime_t hold_caps_until;
+  using time = ceph::coarse_mono_time;
```

Comment:
This should go at the top of the definition for `struct Inode`. See

https://github.com/ceph/ceph/blob/master/CodingStyle

and

https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
Done